config/main.py

```diff
@@ -1715,7 +1715,7 @@ def warm_restart_enable(ctx, module):
 @click.pass_context
 def warm_restart_neighsyncd_timer(ctx, seconds):
     db = ctx.obj['db']
-    if seconds not in range(1,9999):
+    if seconds not in list(range(1,9999)):
```

Comment:
>list [](start = 22, length = 4)

Old code is good for python3

Reply:
True. Looks like 2to3 was a bit overzealous with `range()`. Removed this and others in latest commit.